.ci/setup_env_fedora.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+#
+# Copyright (c) 2017-2018 Intel Corporation
```

Comment:
This is a new file right? I think here is just need 2018 @grahamwhaley  ?

Reply:
A new file should just be 2018. If you copied the file from another (maybe another distro file), and then modified it, then 2017-2018 is fine.